### ISSUE TITLE
Page break between two widgets not working

### ISSUE THREAD

**Issue by LittleHamster** [siteorigin/siteorigin-panels#390]
I don't know if this is a known issue but I couldn't find anything in the issues so I opened up this issue. If you have a layout and you try to insert `<!--nextpage-->` you will get all contents in all pages.

For example create a row with SiteOrigin Editor widget. 
```
content of page 1
<!--nextpage-->
content of page 2
```
you will see the page navigation but both contents are visible in all pages. 

This method also does not work if you want to do a page break between two widgets
```
[SiteOrigin Image] // should be shown on page 1
[SiteOrigin Editor] // editor has <!--nextpage-->
[SiteOrigin Image] // should be shown on page 2
```

**Comment by gregpriday** [comment]
This is a known issue, but it's possible it will work caching mode enabled.
This filters post content much earlier, so might be compatible. I haven't
checked yet though.

**Comment by LittleHamster** [comment]
@gregpriday I do have that option enabled, but still not working. 
also there is an issue reported [here](https://siteorigin.com/thread/page-builder-and-next-page-issue/) 2 years ago. Can this issue be on the road map to be fixed in the next release?

**Comment by gregpriday** [comment]
Yeah, as I mentioned this is a known issue. We do welcome pull requests if
you'd like to take a look at implementing it.

**Comment by AlexGStapleton** [comment]
Duplicate of #290
